<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1805</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPU Scheduling</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font: 12pt &quot;楷体&quot;;
color:black;
}
QFrame{
border:1px solid black;
}
QLineEdit{
border:1px solid black;
}
QTabWidget{
border:1px solid black;
}
QTabBar::tab:selected,QTabBar::tab:hover{
background-color:rgb(250, 255, 198);
}
QListWidget{
border:1px solid black;
font: 16pt &quot;Times New Roman&quot;;
}
QListWidget::item{
border:1px solid black;
font: 16pt &quot;Times New Roman&quot;;
}
QLabel{
border:0px;
}
QPushButton{
border:1px solid black;
background-color:rgb(250, 255, 198);
}
QPushButton:pressed{
padding-left:3px;
padding-top:3px;
}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1771</width>
      <height>871</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>优先权调度</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>15</y>
        <width>951</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="4">
        <widget class="QLabel" name="label_10">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>86</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>86</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Times New Roman&quot;;</string>
         </property>
         <property name="text">
          <string>Running</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="4">
        <widget class="QListWidget" name="RunningPCB">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>86</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>86</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget::item {
height: 86px;
font: 16pt &quot;Times New Roman&quot;;
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="4">
        <widget class="QPushButton" name="btnWaiting">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>等待</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>120</y>
        <width>951</width>
        <height>2</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>135</y>
        <width>951</width>
        <height>691</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="3" colspan="5">
        <widget class="QListWidget" name="SuspendPCB">
         <property name="minimumSize">
          <size>
           <width>310</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget::item {
height: 105px;
font: 16pt &quot;Times New Roman&quot;;
}</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QListWidget" name="ReadyPCB">
         <property name="minimumSize">
          <size>
           <width>310</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget::item {
height: 105px;
font: 16pt &quot;Times New Roman&quot;;
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Times New Roman&quot;;
color:blue;</string>
         </property>
         <property name="text">
          <string>Waiting</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Times New Roman&quot;;
color:red;</string>
         </property>
         <property name="text">
          <string>Suspend </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Times New Roman&quot;;
color:green;</string>
         </property>
         <property name="text">
          <string>Ready</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnDeWaiting">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>结束等待</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QListWidget" name="WaitingPCB">
         <property name="minimumSize">
          <size>
           <width>310</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget::item {
height: 105px;
font: 16pt &quot;Times New Roman&quot;;
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" colspan="4">
        <widget class="QPushButton" name="btnDeSuspend">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>解挂</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>371</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btnAddProcess">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>190</y>
         <width>161</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>添加进程</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>351</width>
         <height>165</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>PID：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>要求运行时间：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>优先权：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>需要内存：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="pcb_name">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pcb_time">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pcb_vip">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pcb_memory">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>392</width>
        <height>821</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>内存条</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="PCBlist">
         <property name="minimumSize">
          <size>
           <width>390</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>390</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>16</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget::item {
height: 85px;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSuspend">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>挂起</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string/>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
